config/post-install/100-istio-default-domain.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: default-domain
+  namespace: knative-serving
+  labels:
+    app: "default-domain"
+    networking.knative.dev/provider: "istio"
```

Comment:
@mattmoor let me know if you would prefer a different label here.

Reply:
This was mainly relevant in the context of the other file renames that you reverted.  I'd label all of the istio stuff holistically as we plan to split off that implementation.